src/app/routes.py

```diff
@@ -125,21 +127,25 @@ def generate_feed_xml(feed: Feed) -> Any:
         lastBuildDate=datetime.datetime.now(),
         items=items,
     )
-    logging.info(f"XML generated for feed with ID: {feed.id}")
+    logger.info(f"XML generated for feed with ID: {feed.id}")
     return rss_feed.to_xml("utf-8")
 
 
 @main_bp.route("/v1/feed", methods=["POST"])
 def add_feed() -> flask.Response:
-    data = request.get_json()
+    if request.is_json:
```

Comment:
nit: how hard would it be to unify it so that all requests were of the same type?

Reply:
good call, we don't need the JSON 👍